src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
+/* eslint-disable no-console */
 'use strict';
 
-// write code here
+const tree = document.querySelectorAll('li');
+
+for (const item of tree) {
+  const header = item.firstChild.textContent;
+
+  item.firstChild.textContent = '';
+
+  item.insertAdjacentHTML(
+    'afterbegin',
+    `<span>${header}</span>`
+  );
+
+  item.firstChild.addEventListener('click', () => {
+    if (item.children[1].style.display !== 'none') {
+      item.children[1].style.display = 'none';
+
+      return;
+    }
+
+    item.children[1].style.display = 'initial';
```

Comment:
```suggestion
    const itemVisibility = item.children[1].style.hidden;

    item.children[1].style.hidden = !itemVisibility;
```

Reply:
Your offer doesn`t work, but I figure out the issue. It must be `block` display instead of `initial`.